src/main.rs

```diff
@@ -335,7 +335,7 @@ async fn main() -> anyhow::Result<()> {
     let hardhat = HardhatNamespaceImpl::new(node.get_inner());
 
     let threads = build_json_http(
-        SocketAddr::new(IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)), opt.port),
+        SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), opt.port),
```

Comment:
Should we make this configurable similar to `opt.port`?

Reply:
Not sure if it will be useful since it's for local setup only and making it 0.0.0.0 already allows access on all urls: localhost, 127.0.0.1, 0.0.0.0, etc.